src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/payment/SendReportTask.java

```
@@ -0,0 +1,94 @@
+    public static final String ATTACHMENT_PREFIX_2 = "Bulk-Scan-Orchestrator-Update-Payment-Daily-Report-";
+    private final JavaMailSender mailSender;
+    private final PaymentService paymentService;
+    private final UpdatePaymentService updatePaymentService;
+    private final String from;
+    private final String[] recipients;
+
+
+    public SendReportTask(
+        JavaMailSender mailSender,
+        PaymentService paymentService,
+        UpdatePaymentService updatePaymentService,
+        @Value("${spring.mail.username}") String from,
+        @Value("${reports.recipients}") String[] recipients) {
+        this.mailSender = mailSender;
+        this.paymentService = paymentService;
+        this.updatePaymentService = updatePaymentService;
+        this.from = from;
+
+        if (recipients == null) {
+            this.recipients = new String[0];
+        } else {
+            this.recipients = Arrays.copyOf(recipients, recipients.length);
+        }
+
+        if (this.recipients.length == 0) {
+            log.warn("No recipients configured for reports");
+        }
+    }
+
+    @Scheduled(cron = "${reports.cron}")
+    public void send() {
+        try {
+            log.info("Sending daily report: {}", EMAIL_SUBJECT);
+            MimeMessage msg = mailSender.createMimeMessage();
+
+            MimeMessageHelper helper = new MimeMessageHelper(msg, true);
+            helper.setFrom(from);
+            helper.setTo(this.recipients);
+            helper.setSubject(EMAIL_SUBJECT);
+            helper.setText(EMAIL_BODY);
+            helper.addAttachment(ATTACHMENT_PREFIX_1 + LocalDate.now() + ".csv", getCreatePaymentCsvReport());
+            helper.addAttachment(ATTACHMENT_PREFIX_2 + LocalDate.now() + ".csv", getUpdatePaymentCsvReport());
+            mailSender.send(msg);
+
+        } catch (Exception exc) {
+            log.error("Error sending report", exc);
```

Should we raise an exception here if there is a problem? Depends if the other report tasks on the other services do this in the same way as note, feel free to let me know